src/cocoa/toga_cocoa/fonts.py

```diff
@@ -20,13 +20,19 @@ def __init__(self, interface):
         try:
             font = _FONT_CACHE[self.interface]
         except KeyError:
+            # Default system font size on Cocoa is 12pt
+            if self.interface.size == -1:
```

Comment:
You use -1 as the default font size in many places. I think it would be good to save this as constant named `DEFAULT_FONT_SIZE` and use it in the various places it is needed. Maybe this constant actually belongs in *Travertino*.
In this way you won't use -1 directly all over your code.

Reply:
Yeah - I wasn't sure about this one myself. 

I contemplated using an `INITIAL` or `SYSTEM` constant; however, that would modifying Travertino (as Travertino requires that font sizes are integers). 

However, on second thought, even using `SYSTEM_FONT_SIZE = -1` as a Pack-specific constant would be a better option than just hard coding -1.